src/framework/level1.js

```diff
@@ -34,6 +28,7 @@ var FrameworkLevel1 = GObject.registerClass({
         model.accent_color = this._accentColorButton.rgba;
         model.info_color = this._infoColorButton.rgba;
         model.logo_color = this._logoColorButton.rgba;
+        model.logo_graphic = 'dinosaur';
```

Comment:
Should this not use `this._logoGroup.value`?

Reply:
Good point.

Reply:
It is fixed, maybe you are looking at an old commit?

Reply:
Ah, you fixed it in the next commit.